File: Programmers/JAVA/lv1/lv1_문자열 다루기 기본.java
https://programmers.co.kr/learn/courses/30/lessons/12918

import java.util.*;

class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        
        if(s.length() == 4 || s.length() == 6){
            for (int i = 0; i < s.length(); i++) {
                if (s.charAt(i) >= '0' && s.charAt(i) <= '9') {
                    answer = true;
                }
                else {
                    answer = false;
                    break;
                }
            }      
        }
        else answer = false;
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_문자열 다루기 기본.cs
https://programmers.co.kr/learn/courses/30/lessons/12918?language=csharp

public class Solution
{
    public bool solution(string s)
    {
        bool answer = true;
        int n = 0;
        answer = (!(s.Length == 4 || s.Length == 6)) ? false : int.TryParse(s, out n) ? true : false;
        //TryParse() 기본 리턴 타입은 boolean 이지만 out으로 변환 가능
        return answer;
    }
}